src/types.ts

```diff
@@ -46,6 +50,11 @@ export type Metadata = {
   translations: TranslationReference[]
 }
 
+export type MetadataDocument = SanityDocument & {
```

Comment:
I think this should use the SanityDocumentLike type instead, which only requires the `_id` and `_type`

Reply:
Great input, will fix!